packages/editor/src/components/rich-text/tinymce.js

```diff
@@ -243,6 +243,15 @@ export default class TinyMCE extends Component {
 
 					// Restore the original `setHTML` once initialized.
 					editor.dom.setHTML = setHTML;
+
+					// In IE11, focus is lost to parent after initialising
+					// TinyMCE, so we have to set it back.
+					if (
+						document.activeElement !== this.editorNode &&
+						document.activeElement.contains( this.editorNode )
```

Comment:
This is a bit concerning. I wonder for example how this works for quotes where we have two RichTexts that might be inside the selection. Can't we check the isFocused element before the `init` call and only focus if it's the current one?

Reply:
I'm not sure what you mean. Could you elaborate?

Reply:
This commit is more restrictive than the last.

Reply:
I'm suggesting doing this before the init call.

```
const wasFocused = document.activeElement === this.editorNode
```

And changing this condition to

```
wasFocused &&
document.activeElement !== this.editorNode &&
document.activeElement.contains( this.editorNode )
```

Reply:
Basically making it a little bit more strict

Reply:
There's no point in checking that. The initialisation is only done on focus, so `document.activeElement === this.editorNode` will always be true.

Reply:
You're totally right :) I forgot that part.

It's still a bit concerning that if we move the focus quickly between two RichTexts in the same block (quote block for instance) while the initialization is in progress, we'll revert the focus. But I guess it's very rare.

Reply:
Yes, that's concerning to me too, which is why I tried some other things to fix it. Don't know what would work though. The best thing would be to figure out why focus is lost and prevent that. Another thing we can do is just not initialise TinyMCE in IE11, but perhaps it's better to wait a bit with that.

Reply:
CAn we test that we're just in IE11 to do that?

Reply:
Yeah, we can detect IE and not initialise. But maybe it's better to wait for the list commands (indent and change type) to work with the rich text value, otherwise they won't work.

Reply:
I was thinking detect if IE and apply the current patch only in IE. (still initialize the editor) just to make sure we avoid side effects in other browsers.